curvefs/src/client/warmup/warmup_manager.h

```diff
@@ -66,18 +66,25 @@ using curvefs::client::common::WarmupStorageType;
 
 class WarmupFile {
  public:
-    explicit WarmupFile(fuse_ino_t key = 0, uint64_t fileLen = 0)
-        : key_(key), fileLen_(fileLen) {}
+    explicit WarmupFile(fuse_ino_t key = 0,
+                        uint64_t fileLen = 0,
+                        const std::string& mountPoint = "",
+                        const std::string& root = "")
+        : key_(key), fileLen_(fileLen), mountPoint_(mountPoint), root_(root) {}
 
     fuse_ino_t GetKey() const { return key_; }
     uint64_t GetFileLen() const { return fileLen_; }
+    std::string GetMountPoint() const { return mountPoint_; }
+    std::string GetRoot() const { return root_; }
     bool operator==(const WarmupFile &other) const {
         return key_ == other.key_;
     }
 
  private:
     fuse_ino_t key_;
     uint64_t fileLen_;
+    std::string mountPoint_;
+    std::string root_;
 };
 
 using WarmupFilelist = WarmupFile;
```

Comment:
Mountpoint and root should be members of warmupfileList?
These two parameters should be unnecessary for warming up a single file.

Reply:
Rather than creating an alias for `WarmupFilelist` with `WarmupFile` class, a new class `WarmupFilelist` is created that inherits from the `WarmupFile` class.